Formula/rav1e.rb

```diff
@@ -5,8 +5,8 @@ class Rav1e < Formula
   head "https://github.com/xiph/rav1e.git", branch: "master"
 
   stable do
-    url "https://github.com/xiph/rav1e/archive/v0.6.5.tar.gz"
-    sha256 "a0d137aa40a27b308f965c4bc03a13493f6d07c888d6b52cc2ffaba36bfd5988"
+    url "https://github.com/xiph/rav1e/archive/v0.6.6.tar.gz"
+    sha256 "723696e93acbe03666213fbc559044f3cae5b8b888b2ddae667402403cff51e5"
 
     # keep the version in sync
     resource "Cargo.lock" do
```

Comment:
We also need to bump Cargo.lock. See for reference, [my patch](https://github.com/barrbrain/homebrew-core/compare/master..rav1e-0.6.6).